src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -65,41 +65,6 @@ message Constraints {
   string expression = 2;
 }
 
-// To reference an image, a user has to provide an image name and a version. A
-// user may specify a version either with
-// a tag value (for example 'latest') or a digest. When submitting a job, a user
-// should provide either a tag or a digest value only (not both of them).
-//
-// For example, docker images can be referenced by {name=titus-examples,
-// tag=latest}. A user could also choose to provide only the digest without a
-// tag. In this case, the tag value would be empty.
-message Image {
```

Comment:
Moving existing types would be a breaking change, at least at the import level, but also generated Java code may land in different packages/classes. I would limit this refactoring to the very recent changes.

Reply:
Are you sure it is a breaking change, as long as I make the JavaClass thing the same as before?

I won't be able to move out these BasicContainer types out without also moving Image, as they depend on Image too.
Or I'll have to copy `Image`

Reply:
I would double check before doing this change. You should see how it will impact generated Java/Python/Go clients. Hopefully there is also some documentation that covers this case.

Reply:
Confirmed using `localIDL` on my branch.

For java, `option java_package = "com.netflix.titus.grpc.protogen";` ensures that they stay in the same package place, so the java class names are all in the same place, just generated from different source .proto files.

For golang, this seems to also work fine, because (at least with titus-executor, and I'm sure other go codebases must do this too), they are already loading titus_base to get the object for generation. Similar deal, the Image go code moves from `titus_job_api.pb.go` to `titus_base.pb.go`, but they are in the same go package (the exact package name is left up to the repo, for titus-executor it ends up as `api/netflix/titus`). 

You are right though that Python will break. 

```
from netflix.titus.titus_job_api_pb2 import JobDescriptor, Image, TaskStatus
```

I think this type of line would need to change to `from netflix.titus.titus_base_pb2 import Image` if we moved it here.

Reply:
I've put this `Image` back into the job_api proto to prevent this breaking change.